File: Composite/Python/Exodia.py
import this
from Carta import *
from Torso import *
from PiernaIzquierda import *
from PierndaDerecha import *
from BrazoDerecho import *
from BrazoIzquierdo import *

class Exodia(Carta):
   # _exodiaParts = []

    def __init__(self):
        self._exodiaParts = []
        self._exodiaParts.append(Torso())
        self._exodiaParts.append(PiernaDerecha())
        self._exodiaParts.append(PiernaIzquierda())
        self._exodiaParts.append(BrazoDerecho())
        self._exodiaParts.append(BrazoIzquierdo())

    def logic(self):
        print(self._exodiaParts)
        for i in range(0, 5):
            self._exodiaParts[i].logic()

File: Composite/Java/Exodia.java
package Java;

import java.util.ArrayList;
import java.util.List;

public class Exodia implements Carta {
    private List<Carta> exodiaParts;

    Exodia()
    {

        exodiaParts = new ArrayList<>();
        
        exodiaParts.add(new Torso());
        exodiaParts.add(new PiernaDerecha());
        exodiaParts.add(new BrazoIzquierdo());
        exodiaParts.add(new PiernaIzquierda());
        exodiaParts.add(new BrazoDerecho());
 
    }

    @Override
    public void logic() {
        
        for(Carta pieza: exodiaParts){
            pieza.logic();
        }
        
    }
}